### DISCUSSION TITLE
tvOS support

### DISCUSSION THREAD

**Discussion by exilef** [Tencent/libpag#2170] [Ideas] [answered]
Hi, it would be great to see the addition of tvOS support - any plans for adding this?

**Comment by kevingpqi123** [comment, marked as answer]
If your tvOS system is based on Android, the aar file for mobile platforms already supports running on TVs. If you are referring to Apple's TVOS, there are currently no plans for adaptation as we do not have a use case. However, PAG is open-source, and you can adapt it based on the open-source code.

**Reply by exilef** [reply]
Yes, I meant Apple tvOS, which is a subset of iOS - sorry for not being specific enough. Are there any plans to start looking into supporting it? It seems to me that the changes needed are not that big, given that tvOS is a subset of iOS. But this is based on a superficial look from the outside, and I am not sure if there are issues with compiling the dependencies for tvOS (such as tgfx, but also others).

**Reply by kevingpqi123** [reply]
You understand correctly that the adaptation work here should not be too extensive. However, due to our limited resources and lack of such use cases, we currently do not have plans to adapt for tvOS. If you need to use it, you may need to adapt it based on the source code.

**Reply by exilef** [reply]
Totally understand, thank you for the quick reply!
